app/javascript/app/providers/climate-plans-provider/climate-plans-provider-actions.js

```diff
@@ -0,0 +1,25 @@
+import { createAction, createThunkAction } from 'redux-tools';
+import { INDOAPI } from 'services/api';
+
+export const fetchClimatePlansInit = createAction('fetchClimatePlansInit');
+export const fetchClimatePlansReady = createAction('fetchClimatePlansReady');
+export const fetchClimatePlansFail = createAction('fetchClimatePlansFail');
+
+export const fetchClimatePlans = createThunkAction(
+  'fetchClimatePlans',
+  isoCode => (dispatch, state) => {
+    const { climatePlans } = state();
+    if (climatePlans && !climatePlans.loading) {
+      dispatch(fetchClimatePlansInit());
+      INDOAPI
+        .get('province/climate_plans', { location: isoCode })
```

Comment:
You can get climate plans by locale. The date is there already.

Reply:
👍